### PR TITLE
Feature/login

### PR DESCRIPTION
**Pull request by StrongCurrent**
Here comes the login. I am happy about a green tick 😀 but also about feedback and suggestions.

### CODE REVIEW

File: components/Nav/index.js [hunk 1]
```diff
@@ -1,26 +1,40 @@
...
     },
+    {
+      id: 3,
+      title: "PROFILE",
+      href: "/profile",
+      icon: PiUserCircleBold,
+      ariaLabel: "Navigate to your profile",
+    },
   ];
+
```
Comment (Roland-Hufnagel) [inline components/Nav/index.js:37]: Very nice solution for creating the navigation! 👍

**Review by Roland-Hufnagel — APPROVED** [review]
Hey Martin. Good integration of next-auth 🏄🚀🏄!!!
I'm just wondering, that I can't create a new location in the map when creating a new spot. Your long and lat is hard-coded since now. Hope this will change in the future.

**Comment by StrongCurrent** [comment]
> Hey Martin. Good integration of next-auth 🏄🚀🏄!!!
> I'm just wondering, that I can't create a new location in the map when creating a new spot. Your long and lat is hard-coded since now. Hope this will change in the future.

Hey Roland thanks for your review. What do you mean you can't create a spot, it works fine for me. Do you want to change the coardinates of an existing spot? That's not planned at the moment, but it's a imaginable extension for the future. The hard-coded longitude and latitude are actually just the initial coordinates of the marker. If you're on the create a new spot page, you can grab the marker and place it anywhere you want. you can't do that with a created spot. Usually the spot doesn't move that far away either ;). Please check again and let me know if it works for you. THX

File: components/Nav/index.js (hunk 1 above)
Comment (StrongCurrent) [reply components/Nav/index.js:37]: THX

**Comment by Roland-Hufnagel** [comment]
Hey Martin. It works! Don't know what went wrong but tried it again and everything is fine!

**Pull request merged**